### PR TITLE
Add licenses and versions to third-party plugins

### PR DESCRIPTION
**Pull request by adamrusted**
Further to internal discussion, PR adds which version of Simple Icons a third-party extension uses, as well as a deprecation message for those developers regarding removal as of v11 if packages not kept up-to-date.

### CODE REVIEW

**Comment by adamrusted** [comment]
Query remains as to whether we should remove everything <v8.0.0 with the v10 release this weekend?

**Comment by LitoMore** [comment]
I would say no. It's really hard to keep the version number update.

**Comment by adamrusted** [comment]
> I would say no. It's really hard to keep the version number up to date.

It's easily done by either querying the package.json on the main branch of our project after a Sunday release, or using a CDN.
Almost all of the projects showing as 'Latest' in this PR use one of those two methods.

File: README.md [hunk 1]
```diff
@@ -149,34 +149,37 @@ echo file_get_contents('path/to/package/icons/simpleicons.svg');
...
+| [Vue package](https://github.com/mainvest/vue-simple-icons) <img src="https://cdn.simpleicons.org/vuedotjs/000/fff" alt="Vue" align=left width=24 height=24> | [@noahlitvin](https://github.com/noahlitvin) | ![](https://img.shields.io/github/license/mainvest/vue-simple-icons) | [v4.7.0](https://github.com/simple-icons/simple-icons/tree/4.7.0) |
+| [WordPress plugin](https://wordpress.org/plugins/simple-icons/) <img src="https://cdn.simpleicons.org/wordpress/000/fff" alt="WordPress" align=left width=24 height=24> |  [@tjtaylo](https://github.com/tjtaylo) | ![](https://img.shields.io/badge/license-GPL_v2-blue.svg) | [v4.25.0](https://github.com/simple-icons/simple-icons/tree/4.25.0)
+
+> **Note for Extension Authors**\
+> As of v11 we will be removing third-party extensions from the above list that do not keep up to date with the previous major release. For example, when v11 releases we will remove anything not supporting `>v9.0.0`.
```
Comment (PeterShaggyNoble) [inline README.md:182]: I'd reword this slightly to say:
> From our next major release (v11 in [Insert planned month of release]), we will begin removing third-party extensions from the above list that are not up to date with at least our previous major release. For example, when v11 is released, we will remove any extensions that don't support `v10.0.0` or higher.

And then maybe add:
> Please create a PR to update the version number of your extension in this README following each update of your extension.

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]

**Review by PeterShaggyNoble — APPROVED** [review]

**Comment by LitoMore** [comment]
> It's easily done by either querying the package.json.

It's not easy. For example this package https://github.com/avmaisak/ngx-simple-icons.

The version number is not in their package.json.

I prefer to revert this PR due to it's hard to maintain.

**Comment by adamrusted** [comment]
We can assist third-party maintainers if they request it, or if we've noticed they've been inactive for some time. For example, I've submitted a solution to the [hexo plugin](https://github.com/nidbCN/hexo-simpleIcons/pull/1) to bring it up to scratch. Others will (and have already) either mimic the methods we use for the website repositories, or switch to a `@latest` tag when importing our repository.

Once we know (as of v11 release) that all 3rd party plugins use either `@latest` or build weekly following our releases,, we can revert the table to just the package name and author - but I think it's vital we point out to users that these packages are using an older (and thus, unmaintained) version of the package.
